packages/design-system/src/components/web-components/ds-date-field/ds-date-field.stories.tsx

```diff
@@ -0,0 +1,68 @@
+import { useEffect } from 'react';
+import { action } from '@storybook/addon-actions';
+import type { Meta } from '@storybook/react';
+import WebComponentDocTemplate from '../../../../../../.storybook/docs/WebComponentDocTemplate.mdx';
+import { webComponentDecorator } from '../storybook';
+import './ds-date-field';
+
+const meta: Meta = {
+  title: 'Web Components/ds-date-field',
+  decorators: [webComponentDecorator],
+  parameters: {
+    docs: {
+      page: WebComponentDocTemplate,
+    },
+  },
+  args: {
+    hint: 'If you were born on a leap day, entering the date will either crash our servers or open a portal to an alternate dimension.',
+    label: 'Enter your date of birth.',
+    name: 'ds-date-field',
+  },
+  argTypes: {
+    errorMessage: { control: 'text' },
+    hint: { control: 'text' },
+    requirementLabel: { control: 'text' },
```

Comment:
I'm not sure that `requirementLabel` is wired up correctly. I was not able to see the label change after updating this line from `requirementLabel` to `requirement-label` and then in storybook include a value for that prop.

Reply:
Looks like adding `'requirement-label'` to the array of attributes found in `packages/design-system/src/components/web-components/ds-date-field/ds-date-field.tsx` is needed